pipeline/inputs/windows-exporter-metrics.md

```diff
@@ -18,6 +18,9 @@ The initial release of Windows Exporter Metrics contains a single collector avai
 | Key             | Description                                                            | Default   |
 | --------------- | ---------------------------------------------------------------------- | --------- |
 | scrape\_interval | The rate at which metrics are collected from the host operating system | 5 seconds |
+| we.logical\_disk.allow\_disk\_regex | Specify to be allowed regex for logical disk metrics                   | "/.+/"    |
+| we.logical\_disk.deny\_disk\_regex  | Specify to be denied regex for logical disk metrics                    | `NULL`    |
+| we.net.allow\_disk\_regex           | Specify to be allowed regex for net metrics by name of NIC             | "/.+/"    |
```

Comment:
I think the setting has a typo here.
Why is there no deny configuration?

Reply:
There is not typo for this. 
For net collector, I didn't implement for denied regex mechanism. Only for allowed regex here:
https://github.com/fluent/fluent-bit/blob/master/plugins/in_windows_exporter_metrics/we_net.c#L213

Reply:
It says `disk_regex` which seems to be a typo - I would suspect it says `we.net.allow_nic_name_regex` or something.

Currently this seems to select all names so presumably you would deny by modifying the regex to exclude them?

Reply:
```suggestion
| we.net.allow\_nic\_name\_regex           | Specify the regex for network metrics captured by the name of the NIC, by default captures all NICs but to exclude adjust the regex. | "/.+/"    |
```

Reply:
Ah, good catch. Actually, I've overlooked for the typo. I'd fixed them in https://github.com/fluent/fluent-bit-docs/pull/1009/commits/f99bff047f845e1e8df04566f9d864adff02fd05.